### ISSUE TITLE
Settings UX: arrow keys do not properly switch focus

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52468]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 2.4.3

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2. Navigate to Settings icon in Activity Bar->Press Enter->Select "Settings" from dropdown.
3. Focus is lost when upward arrow key is pressed(once) while navigating through the list items in a Bottom-Up manner on the screen.

**Actual:**
Keyboard focus is lost when upward arrow key is pressed(once) while navigating through the list items in a Bottom-Up manner.

No action takes place and keyboard focus is found missing when the upward arrow key is pressed once while moving up the list of the items.

**Expected:**
Keyboard focus should not be lost when upward arrow key is pressed(once) while navigating through the list items in a Bottom-Up manner.

Keyboard focus should be  found on the previous list item  when the upward arrow key is pressed once while moving up the list of the items.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The end user will not be able to understand the movement of the keyboard focus if the focus is lost. 

**MAS Reference:**
MAS 2.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachment For Reference:**
[A11y_VSCode_UserSettings_Keyboard_FocusLoss2.pptx](https://github.com/Microsoft/vscode/files/2119413/A11y_VSCode_UserSettings_Keyboard_FocusLoss2.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This is the new settings ux editor, moving to @roblourens
